src/internal/communication.ts

```diff
@@ -312,6 +312,7 @@ function createMessageRequest(func: string, args: any[]): MessageRequest {
   return {
     id: CommunicationPrivate.nextMessageId++,
     func: func,
+    time: new Date().toISOString(),
```

Comment:
Could we just use a simple milliseconds timestamp here instead of a fully formatted date? Formatting the date like this seems to incur a performance hit which even though small would just add latency to our API calls:
![image](https://user-images.githubusercontent.com/3357245/114949792-99aa4f80-9e06-11eb-9125-d6dc340e6a51.png)